core/src/main/java/org/bitcoinj/core/AbstractBlockChain.java

```diff
@@ -774,9 +774,9 @@ private static long getMedianTimestampOfRecentBlocks(StoredBlock storedBlock,
                                                          BlockStore store) throws BlockStoreException {
         long[] timestamps = new long[11];
```

Comment:
I wonder if this should be converted to an array of `Instant`?

Reply:
We could rewrite the entire method at any time, including it's return type. It's private. But again, out of scope for this PR.

Reply:
OK

Reply:
See  #3506